distribution/lib/Standard/Base/0.0.0-dev/src/Data/Decimal.enso

```diff
@@ -1342,6 +1343,13 @@ type Decimal
     to_display_text : Text
     to_display_text self -> Text = self.big_decimal.toString
 
+    ## ---
+       private: true
+       ---
+    to_js_object : JS_Object
```

Comment:
Feel like this waa handled somewhere already? Maybe as an extension method.

Reply:
Yep. Removed.